test/hpke_test.c

```diff
@@ -1741,8 +1741,39 @@ static int test_hpke_ikms(void)
 
     return res;
 }
+typedef enum OPTION_choice {
+    OPT_ERR = -1,
+    OPT_EOF = 0,
+    OPT_VERBOSE,
+    OPT_TEST_ENUM
+} OPTION_CHOICE;
+
+const OPTIONS *test_get_options(void)
+{
+    static const OPTIONS test_options[] = {
+        OPT_TEST_OPTIONS_WITH_EXTRA_USAGE("conf_file\n"),
```

Comment:
run the test from command line to see the usage output...

Reply:
yep, fixed now I think